docs/source/dashboard.rst

```diff
@@ -61,27 +80,53 @@ indicate the following.
 
     <table>
         <tr>
+            <th colspan="2">For individual worker bars</th>
```

Comment:
This part seems a bit repetitive and I think we want to avoid mentioning specific numbers like 250MiB. 

I think we can consolidate everything in column. 

This is how it renders. 
![Screen Shot 2023-01-09 at 2 36 45 PM](https://user-images.githubusercontent.com/7526622/211394702-491ae5dc-3112-4b3b-9dd7-ac25206fffa0.png)

Reply:
Updated it as written the description in only one column only, as suggested.